packages/manager/modules/pci/src/projects/project/ai/apps/components/delete/translations/Messages_fr_FR.json

```diff
@@ -1,8 +1,8 @@
 {
-  "pci_ai_apps_delete_success": "Votre app a été supprimée avec succès",
-  "pci_ai_apps_delete_error": "Une erreur est survenue durant la suppression de l'app : {{ message }}",
-  "pci_ai_apps_delete": "Suppression de l'app",
+  "pci_ai_apps_delete_success": "Votre AI app a été supprimée avec succès",
+  "pci_ai_apps_delete_error": "Une erreur est survenue durant la suppression de l'AI App : {{ message }}",
+  "pci_ai_apps_delete": "Suppression de l'AI app",
   "pci_ai_apps_delete_common_delete": "Supprimer",
   "pci_ai_apps_delete_common_cancel": "Annuler",
-  "pci_ai_apps_delete_warning": "Êtes-vous sûr(e) de vouloir supprimer l'app {{ appId }} ?"
```

Comment:
To be consistent with the rest of content we could use `sûr·e` instead of `sûr(e)`.

Reply:
done